dashboard/app/mailers/pd/application/teacher1819_application_mailer.rb

```diff
@@ -11,6 +11,39 @@ def confirmation(teacher_application)
       )
     end
 
+    def accepted(teacher_application)
+      raise "Unexpected #{teacher_application.class}" unless teacher_application.is_a? Teacher1819Application
+
+      @application = teacher_application
+
+      mail(
+        to: @application.user.email,
+        subject: "You've been accepted to Code.org's Professional Learning Program!"
+      )
+    end
+
+    def declined(teacher_application)
+      raise "Unexpected #{teacher_application.class}" unless teacher_application.is_a? Teacher1819Application
+
+      @application = teacher_application
+
+      mail(
+        to: @application.user.email,
+        subject: "Update on your Code.org Professional Learning Program application"
+      )
+    end
+
+    def waitlisted(teacher_application)
+      raise "Unexpected #{teacher_application.class}" unless teacher_application.is_a? Teacher1819Application
```

Comment:
nit: as with above, we probably don't need these checks. Initially I had them on the other application emails and removed them as unnecessary on another PR review, since these email methods are only called in one place in the controller or cronjob, for the correct type.